.cursor/rules/tracker-allowlist.mdc

```diff
@@ -0,0 +1,337 @@
+---
+globs: features/tracker-allowlist.json,overrides/**/*.json
+alwaysApply: false
+---
+
+# Tracker Allowlist Mitigation Instructions
+
+## Background
+
+Tracker Allowlist is used to allow specific tracker requests that would normally be blocked by the tracker radar, where necessary to fix website breakages. The feature configuration lists rules of which tracker requests should be allowed for which websites. Care must be taken when writing these rules, since allowing tracker requests reduces privacy protection for users.
+
+See:
+- [Tracker Allowlist rule definitions](/features/tracker-allowlist.json)
+  (Applies to all browser platforms.)
+- [Platform-specific configuration override files](/overrides/)
+- [The schema for tracker allowlist rules](/schema/features/tracker-allowlist.ts)
+- [Bugbot PR review guidelines](/cursor/BUGBOT.md)
+
+## Table of Contents
```

Comment:
I'm not sure this is useful. I don't think an .mdc file will auto-format like an .md file on github anyway, so even the humans won't be able to make use of these hashtag shortcuts (e.g., `#11-not-too-generic-nor-too-specific`). I'd just zap this and any of the hash links in the rest of the body (won't break anything, but adds noise).